htdocs/luci-static/resources/view/homeproxy/client.js

```diff
@@ -313,6 +313,15 @@ return view.extend({
 		so.depends('tcpip_stack', 'gvisor');
 		so.rmempty = false;
 
+		so = ss.option(form.Value, 'udp_timeout', _('UDP NAT expiration time'),
+			_('In seconds. <code>5m</code> is used by default.'));
+		so.datatype = 'uinteger';
+		so.default = '300';
```

Comment:
```suggestion
		so.placeholder = '300';
```

since it cannot be empty. tbh do we really need to specify a preset value instead of leaving it default (empty)?

Reply:
keep empty is not bad. i will modify it